app/templates/buyers/start_brief_info.html

```diff
@@ -76,98 +73,48 @@ <h2 class="au-display-lg">After your brief closes:</h2>
-          <li>
-            To help prepare your brief offline, 
-            <a href="/static/media/documents/training_brief_draft_questions.docx">
-              download the draft questions (DOCX 391KB)
-            </a>.
-          </li>
-          <li>
-            For a step by step guide, please read the 
-            <a href="https://marketplace1.zendesk.com/hc/en-gb/articles/360000299476">
-              training guide
-            </a> or 
-            <a href="/contact-us">contact us</a>.
-          </li>
-          <li>
-            If you need an introduction in delivering digital services, agile delivery, 
-            user research or content design, use the 
-            <a href="https://govau.github.io/learning-design-standards/">
-              learning design standards
-            </a> to help structure your brief.
-          </li>
-        </ul>
-        <h2 class="au-display-lg">Writing your brief:</h2>
-        <ul>
-          <li>
-            Try not to be too specific in your training requirements. This will help sellers 
-            define the best training approach for you.
-          </li>
-          <li>You can contact sellers to discuss your requirements before publishing.</li>
-        </ul>
-        <h2 class="au-display-lg">While your brief is live:</h2>
-        <ul>
-          <li>
-            If sellers ask questions, the person who published the brief will receive them via email. 
-            Make sure they are available and can 
-            <a href="https://marketplace1.zendesk.com/hc/en-gb/articles/115011257447-Answer-questions">
-              respond
-            </a> within the required timeframes.</li>
-        </ul>
-        <h2 class="au-display-lg">After your brief closes:</h2>
-        <ul>
-          <li>You will receive seller responses, including proposals, costs and résumés after 1 to 2 weeks.</li>
-          <li>Evaluate the responses using the criteria you set when writing the brief.</li>
-          <li>
-            <a href="https://marketplace1.zendesk.com/hc/en-gb/articles/115011257587-Award-a-contract">
-              Award the contract
-            </a> to the applicant who best meets your needs and represents value for money.</li>
+          <li>Request proposals from <a href="/search/sellers/?role=Training,%20Learning%20and%20Development&sort_by=a-z&view=sellers&user_role=buyer">approved sellers</a> to build digital skills and expertise.</li>
+          <li>Answer seller questions while your opportunity is live.</li>
+          <li>Download responses once your opportunity has closed.</li>
+          <li>Draft a work order.</li>
         </ul>
+        <p>
+          You can use the <a href="https://govau.github.io/learning-design-standards/" rel="external">learning design standards</a> to help structure your approach.
+        </p>
+        <p>
+          <a href="/static/media/documents/training_brief_draft_questions.docx">
+            Download the draft questions (DOCX 23KB)
```

Comment:
I see a different file size when I check out this branch. 
```suggestion
            Download the draft questions (DOCX 391KB)
```